service/src/main/java/com/myhome/controllers/AmenityController.java

```diff
@@ -18,11 +18,12 @@
 
 import com.myhome.api.AmenitiesApi;
 import com.myhome.controllers.mapper.AmenityApiMapper;
-import com.myhome.model.AmenityDto;
+import com.myhome.model.AddAmenityRequest;
+import com.myhome.model.AddAmenityResponse;
+import com.myhome.domain.Amenity;
 import com.myhome.model.GetAmenityDetailsResponse;
-import com.myhome.model.UpdateAmenityRequest;
+import com.myhome.model.*;
```

Comment:
Wildcards imports.
Why is it bad? Read it here https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

How you can fix it in intellij?
https://stackoverflow.com/a/25916603/5785871